gulpfile.js

```diff
@@ -1106,6 +1121,67 @@ function compileJs(srcDir, srcFilename, destDir, options) {
   }
 }
 
+/**
+ * Given a file path `foo/bar.js`, transpiles the TypeScript entry point of
+ * the same name `foo/bar.ts` and all direct and indirect TypeScript imports.
+ *
+ * @param {string} srcDir
+ * @param {string} srcFilename
+ */
+function transpileTs(srcDir, srcFilename) {
+  const startTime = Date.now();
+
+  const tsEntry = path.join(srcDir, srcFilename).replace('.js', '.ts');
+  const tsConfig = ts.convertCompilerOptionsFromJson({
+    'module': 'ES6',
+    'target': 'ES6',
+  }, srcDir);
+  const tsOptions = tsConfig.options;
+  if (tsConfig.errors.length) {
+    log(colors.red('TSickle:'), tsickle.formatDiagnostics(tsConfig.errors));
+  }
+
+  const compilerHost = ts.createCompilerHost(tsOptions);
+  const program = ts.createProgram([tsEntry], tsOptions, compilerHost);
+
+  const transformerHost = {
+    shouldSkipTsickleProcessing: () => false,
+    transformTypesToClosure: true,
+    options: tsOptions,
+    host: compilerHost,
+  };
+  const emitResult = tsickle.emitWithTsickle(program, transformerHost,
```

Comment:
For debugging purposes it might be beneficial to be able to disable tsickle and use the standard TypeScript compiler directly.

Reply:
Good with me. Awesome work.